### DISCUSSION TITLE
WYSIWYG editor - View Page Source reveals relative links. Possible to configure as absolute?

### DISCUSSION THREAD

**Discussion by jord8on** [requarks/wiki#3452] [Help / Questions]
**Question**
Is there a way to, when clicking "View Page Source," have all of my image and file links appear as absolute links?

We want to be able to select and copy all text from a page (HTML) source, and then paste it in our customer documentation app (different than our Wiki.js repository).

The challenge right now is when we copy the page source HTML, it has a bunch of relative links and when we paste in our other documentation app, naturally all the images are broken.

Is there a toggle I could switch on/off somewhere?

––––––––––––––––––––––––––––––––––––

Side note... after adding an image to a page, I selected it... hit "Copy" and then pasted it at another point in my page. THAT particular image does show the absolute link, while the images I insert from the Asset Manager, all show up as relative links.

––––––––––––––––––––––––––––––––––––

![image](https://user-images.githubusercontent.com/20749549/94849852-48df8600-03e3-11eb-9317-adcd38acf8ef.png)

**Comment by Maxzor** [comment]
Recursing within postgres the asset directory hierarchy would be neat
